_data/social.yml

```diff
@@ -271,14 +271,14 @@ websites:
       img: weibo.png
       tfa: No
 
-    - name: WordPress.com
+    - name: WordPress
```

Comment:
Wordpress.com is different from Wordpress.org and I want us to be explicit about that, so lets keep ".com" part